README.md

```diff
@@ -0,0 +1,247 @@
+//	You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+-->
+
+## What you’ll learn
+
+You will learn how to use the Command Line Interface (CLI) to configure your local development environment to access customized  application stacks. You will then learn how to use the CLI to create, run, test, and debug an application built from an application stack. Finally, you will learn how to build a deployment image that is ready to run in a Kubernetes or serverless environment.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/install/) must be installed.
+- [Appsody](https://appsody.dev/docs/getting-started/installation) must be installed.
+
+## Getting started
+
+Application stacks enable applications to be built and tested inside a container. When built, the container can be deployed into a Kubernetes or serverless environment.
+
+When you install Appsody, the default configuration references the public open source repositories, which contain all the available application stacks from the Appsody project. Although you can use the content in a public repository, an organization typically
+customizes a subset of these application stacks to suit their own requirements. To develop microservice applications from customized application stacks, you must modify your local configuration to point to these stacks before creating a project.
+
+## Discovering repositories and stacks
+
+**CLI command:** `appsody repo list`
+
+To view the current repositories that you have access to, run the `appsody repo list` command. The asterisk shown in the output indicates the default repository.
+
+```
+NAME            URL
+*incubator      https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+experimental    https://github.com/appsody/stacks/releases/latest/download/experimental-index.yaml
+```
+
+By default, your local development environment can access the `incubator` and `experimental` public repositories.
+
+### Adding a customized application stack repository
+
+**CLI command:** `appsody repo add <repo-name> <URL>`
+
+As a developer, your enterprise architect might provide you with a URL that points to a set of customized application stacks for your organization. This configuration is defined by an `index.yaml` file.
+
+To add this configuration information to your local development environment, use the `appsody repo add <repo-name> <URL>` command, supplying a name for the repository and the URL that contains the `index.yaml` file.
+
+In this example, you will add a new repository `abc-stacks` that has a stack configuration URL https://github.com/abc.inc/stacks/index.yaml.
```

Comment:
abc is actually a well-known company in the US?  I think typically we use `acme` ?

Reply:
I've switched the references to "acme" - I don't think there is a precedent, but that's as good as any.